src/Avalonia.Controls/ComboBox.cs

```diff
@@ -184,6 +184,25 @@ protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
             this.UpdateSelectionBoxItem(SelectedItem);
         }
 
+        // Because the SelectedItem isn't connected to the visual tree
+        public override void InvalidateFlowDirection()
```

Comment:
This can be removed because the SelectedItem should be connected to the logical tree and therefore it receives the paren't FlowDirection

Reply:
There was a problem with this before, because the content is not connected to the visual tree (only for the logical time), the `InvalidateFlowDirection` method was not called on the content.

It seems like now it really works without this, I removed it.

Reply:
Okay then let's keep it for now